otherlibs/dune-rpc/private/dune_rpc_private.ml

```diff
@@ -440,14 +440,25 @@ module Message = struct
     iso (record (both payload message)) to_ from
 end
 
-module Subscribe = struct
-  type t =
-    | Diagnostics
-    | Build_progress
+module Sub = struct
+  type 'a t =
+    { diff : 'a Conv.value
```

Comment:
I think the word "diff" presupposes the existence of operations that we don't use and don't need (ability to apply the diffs, or to merge them).
If we're simply implementing lossless streams then we should call this something more neutral, e.g. `elem`.

Reply:
Renamed to elem.